symplyphysics/definitions/mass_fraction_of_the_solute.py

```diff
@@ -0,0 +1,35 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, dimensionless)
+
+# Description
+## The mass fraction is the ratio of the mass of the solute to the total mass of the solution
```

Comment:
I do not like the word `solute` here. Check for more common definition of the mass fraction.

Reply:
> Мне не нравится это слово `solute`здесь. 

And if I replace it with 'mixture'?

Reply:
Mixture is synonim of `solution`. Did you mean `component`?

Reply:
I find basically a similar definition of this term:

> The mass fraction of a substance within a mixture is the ratio of the mass of that substance to the total mass of the mixture.

And I thought to correct the description as follows:
```python
# Description
## The mass fraction of a substance within a mixture is the ratio 
## of the mass of that substance to the total mass of the mixture:
## omega = m_i / m
## Where:
## m_i - mass of component
## m - mass of mixture
## omega - mass fraction of the mixture
```

Reply:
Yes, it's better this way.